cleanup-passes/travis.ts

```diff
@@ -147,5 +147,5 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
 register({
   name: 'travis',
   pass: cleanupTravisConfig,
-  runsByDefault: true,
+  runsByDefault: false,
```

Comment:
Intentional. We never landed all of the travis fixups, which still need some hand-fixing afaik